### ISSUE TITLE
First version and add to Zenodo

### ISSUE THREAD

**Issue by LinguList** [lexibank/daniellinguafrancas#15]
@chrzyki @johenglisch  and @ilchec and @xrotwang, what is left to do here, now that the dataset has been published is:

* add concept list to concepticon (@ilchec, you could follow the guideline and do this yourself now, please, asking @annikatjuka and @schweikhard for advice if needed)
* make a release with the new concepticon version once this has been published

One could make a pre-release without the concept list, to have the data on Zenodo already.

**Comment by LinguList** [comment]
Paper is here: https://muse.jhu.edu/article/806347

**Comment by LinguList** [comment]
@ilchec, could you provide a bibtex version of the paper with full details (number, volume, etc.) here, please?

**Comment by ilchec** [comment]
@LinguList, @chrzyki  I got to the metadata stage successfully and now I'm trying to run `concepticon create_metadata` and keep getting this error:

`ValueError: invalid format for multiple authors: Hays, D. G., Margolis, E., Naroll, R., & Perkins, D. R.`

I tried fixing this entry in `references.bib` and in `conceptlists.tsv`, but it didn't help. 

Can you please help?

@AnnikaTjuka tried to help (and everything worked with my files on her computer), but we couldn't quite figure out what the problem is on my side.

**Comment by xrotwang** [comment]
`Hays, D. G. and Margolis, E. and Naroll, R. and Perkins, D. R.` should work.

**Comment by ilchec** [comment]
@xrotwang nope, still the same error. I change this entry in these two files and just run the script, no other actions after changing them, right? It looks like it doesn't react to the change or maybe there is another file that needs changing?

**Comment by AnnikaTjuka** [comment]
The strange thing is that the entry for Hays et al. was and is correct. @ilchec didn't change it in the first place but just added another concept list.

**Comment by xrotwang** [comment]
@ilchec can you create a PR with your current status of concepticon-data - so we can reproduce the problem?

**Comment by xrotwang** [comment]
AFAIKT, this string is formatted differently in current HEAD of concepticon-data:
```
$ grep -r Margolis concepticondata/
concepticondata/conceptlists.tsv:Hays-1972-11	Hays, David G. and Margolis, Enid and Naroll, Raoul and Perkins, Dale R.	1972		11	colors	English	global languages	http://dx.doi.org/10.1525/aa.1972.74.5.02a00050Hays1972	Hays1972	Eleven focal colors selected to measure the co-evolution of basic color terms and social complexity.	1118	
concepticondata/references/references.bib:  Author                              = {David G. Hays and Enid Margolis and Raoul Naroll and Dale Revere Perkins},
```
So you may have to merge master into your branch?

**Comment by ilchec** [comment]
@xrotwang looks like I can't push my local branch to concepticon, error 403. 

Yes, I thought it's the formatting causing the error, but changing it didn't help.

**Comment by ilchec** [comment]
I think I have an idea what happened, I had the wrong path to concepticon in my configs. I'll try with the new version and see what happens.

**Comment by ilchec** [comment]
It worked!

**Comment by LinguList** [comment]
Okay, so can you make a PR, @ilchec, so we can check this? We'd add the data to Zenodo, once the concepticon data has been released (some time in December, I assume).

**Comment by ilchec** [comment]
@LinguList looks like all branches in this repo have been merged already, so I'm not sure what the PR is supposed to be for. There's only main and two other branches are stale and can be removed, I believe. Or do you mean concepticon? I made a PR there after adding the PDF.

**Comment by chrzyki** [comment]
A PR that changes the code to make use of the concept list in Concepticon instead of a local concept list in `etc/`, i.e. this part: https://github.com/lexibank/daniellinguafrancas/blob/7828efe4fe49c13459576a286c4a82fea5bb2957/lexibank_daniellinguafrancas.py#L53 (compare, e.g. https://github.com/lexibank/halenepal).

Also, the repo could benefit from a removal of things that would belong into `.gitignore` anyway (e.g. `egg-info`, `__pycache__`).

**Comment by chrzyki** [comment]
> 
> 
> A PR that changes the code to make use of the concept list in Concepticon instead of a local concept list in `etc/`, i.e. this part:
> 
> https://github.com/lexibank/daniellinguafrancas/blob/7828efe4fe49c13459576a286c4a82fea5bb2957/lexibank_daniellinguafrancas.py#L53
> (compare, e.g. https://github.com/lexibank/halenepal).
> 
> Also, the repo could benefit from a removal of things that would belong into `.gitignore` anyway (e.g. `egg-info`, `__pycache__`).

Note that code that makes use of the concept list in Concepticon would have to target `concepticon-data` `master` until a new release of Concepticon (as Mattis was describing).

**Comment by ilchec** [comment]
@chrzyki I looked into it and I still have no idea how the file should be changed. Can you help please?

**Comment by chrzyki** [comment]
> 
> 
> @chrzyki I looked into it and I still have no idea how the file should be changed. Can you help please?

Sure! I can either do it for you but if you'd like to do it on your own:

1) Change `metadata.json` `conceptlist` to refer to the correct concept list ID (see here for an example https://github.com/lexibank/othanieljen/blob/master/metadata.json, ID would be `Daniel-2021-160`)
2) Make sure to run `cldfbench lexibank.makecldf` against Concepticon master (i.e. without a version specifier)
3) Remove `concepts.tsv` from `etc/`
4) Map concepts from the concept list like so: https://github.com/lexibank/othanieljen/blob/1262a831e18a3308f175579e7d098cfde8010665/lexibank_othanieljen.py#L21-L23
5) Adapt the parameter ID mapping if need be: https://github.com/lexibank/daniellinguafrancas/blob/7828efe4fe49c13459576a286c4a82fea5bb2957/lexibank_daniellinguafrancas.py#L68

**Comment by ilchec** [comment]
Thanks! Yes, I'd like to do it myself. I'll try what you proposed in a new branch in a couple hours (Have an appointment in 20 mins).
